prac_05/word_occurrences.py

```diff
@@ -0,0 +1,22 @@
+"""
+Word Occurrences
+Estimate: 25 minutes
+Actual: 26 minutes
+"""
+
+word_to_count = {}
+text = input("Text: ")
+
+words = text.split()
+for word in words:
+    try:
+        word_to_count[word] += 1
+    except KeyError:
+        word_to_count[word] = 1
+
+words = list(word_to_count.keys())
+words.sort()
```

Comment:
Quoting from my review, I was suggested to use sorted(). Probably you might want to experiment on the other ways of sorting.